### ISSUE TITLE
panic with some cases

### ISSUE THREAD

**Issue by hogedigo** [knightso/enameg#3]
It would cause panic with cases below.

- only type definition without constants.
- no comment after a constant.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x10dfd7a]

goroutine 1 [running]:
github.com/knightso/enameg.newConst(0xc42009ec90, 0xd, 0xc420078270, 0x0, 0x0, 0x0, 0x0, 0x0)
       /Users/xxxxxx/vendor/src/src/github.com/knightso/enameg/enameg.go:177 +0x12a
github.com/knightso/enameg.Generate(0xc4200c6000, 0x113, 0x1f3, 0x113, 0x1f3, 0x6, 0x0)
       /Users/xxxxxxx/vendor/src/src/github.com/knightso/enameg/enameg.go:79 +0x3e0
main.main()
       /Users/xxxxxxx/vendor/src/src/github.com/knightso/enameg/cmd/enameg/main.go:45 +0xa8
```

**Comment by hogedigo** [comment]
follow-up report.

> * only type definition without constants.

It would not cause panic.
Please just check it.

**Comment by hogedigo** [comment]
one more case.
- double quotation `"` in a constant comment.

**Comment by hogedigo** [comment]
fixed with #5
